Source/WebKitLegacy/mac/WebView/WebPreferences.mm

```diff
@@ -1796,12 +1796,11 @@ - (void)setShowDebugBorders:(BOOL)enabled
 
 - (BOOL)subpixelAntialiasedLayerTextEnabled
```

Comment:
Mind moving this to the `@implementation WebPreferences (WebPrivateObsolete)` section?